go.mod

```diff
@@ -9,6 +9,8 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/pelletier/go-toml/v2 v2.0.6
 	github.com/stretchr/testify v1.8.1
+	github.com/thedatashed/xlsxreader v1.2.5
+	github.com/xuri/excelize/v2 v2.8.0
```

Comment:
we should just keep one dependency of it

Reply:
Done